crates/wasmtime/src/compile/code_builder.rs

```diff
@@ -139,6 +149,53 @@ impl<'a> CodeBuilder<'a> {
         Ok((&wasm[..]).into())
     }
 
+    /// Explicitly specify DWARF `.dwp` path.
+    pub fn dwarf_package_file(&mut self, file: &Path) -> Result<&mut Self> {
+        if self.dwarf_package.is_some() {
+            bail!("cannot call `dwarf_package` or `dwarf_package_file` twice");
+        }
+
+        if !file.exists() {
+            bail!("DWARF package file does not exist");
+        }
+
+        let dwarf_package = std::fs::read(file)
+            .with_context(|| format!("failed to read dwarf input file: {}", file.display()))?;
+        self.dwarf_package = Some(dwarf_package.into());
```

Comment:
In addition to setting `self.dwarf_package`, could this file also set `self.dwarf_package_path`? For example the mutation happening below in `dwarf_package_from_wasm_path` would be pushed into this method. That way if this method is called independently then it'll set `self.dwarf_package_path` automatically.

Reply:
Thanks, done.